woocommerce/woocommerce-gutenberg-products-block
# Move `CheckboxList` and create new `blocksComponents` package

## What

- Adds a `blocksComponents` package. This is added to the `wc` global object.
- Moves the `CheckboxList` component to that package (done in this PR so there's actually something testable there)

## Why

As more developers integrate their extensions with WooCommerce Blocks, the case for reusing components grows. This starts the ball rolling by creating a new package. Rather than recreating components developers can access `wc.blocksComponents` and get the ones we need.

## Testing Instructions



_Please consider any edge cases this change may have, and also other areas of the product this may impact._

1. Add the `Filter by Rating`, `Filter by Attribute`, `Filter by Stock`, and `Products (Beta)` blocks to a page.
2. View the page and ensure all the checkboxes work correctly on each of the filter blocks.
3. View the Cart and then Checkout blocks. Ensure they render correctly.
4. Check out and ensure the Checkout form works correctly (Should not be affected by this PR but run this test anyway to make sure we didn't miss anything)


## Screenshots or screencast



## WooCommerce Visibility


Required:


## Checklist

Required:

Conditional:

## Changelog


> Add a `blocksCheckout` package and export the `CheckboxList` component in it.
